app/views/claims/verified.html.erb

```diff
@@ -5,9 +5,28 @@
     </h1>
 
     <p class="govuk-body">
-      You can now continue with your claim
+      This is your <%= current_claim.presented_identity.verified_fields %> from your digital identity:
     </p>
 
+    <dl class="govuk-summary-list govuk-!-margin-bottom-9">
+      <% [:full_name, :date_of_birth, :payroll_gender, :address].each do |k| %>
+        <% if current_claim.presented_identity.send(k) %>
+          <%= render partial: "answer_row", locals: { label: I18n.t("verified_fields.#{k}").capitalize, answer: current_claim.presented_identity.send(k), slug: nil } %>
+        <% end %>
+      <% end %>
+    </dl>
+
     <%= button_to "Continue", claim_path, method: :put, class: "govuk-button" %>
+
+    <details class="govuk-details" data-module="govuk-details">
+      <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+          Some of this information is wrong
+        </span>
+      </summary>
+      <div class="govuk-details__text">
+        If you find that some of this information is incorrect please contact support at <%= mail_to "studentloanteacherpayment@digital.education.gov.uk", "studentloanteacherpayment@digital.education.gov.uk", class: "govuk-link" %> as this could affect your application.
```

Comment:
Quick question, did we want to use the translation file to save the TSLR specific support address?

Reply:
Good shout 👍

Reply:
we'll want to remember that this will need updating/making dynamic when we introduce Maths & Physics.